.github/workflows/gradle-check.yml

```
@@ -27,6 +27,8 @@ jobs:
       env:
         MAPBOX_KEY: ${{ secrets.MAPBOX_KEY }}
       run: echo mapbox.key="$MAPBOX_KEY" > local.properties
+    - name: Configure server endpoint
+      run: echo server.url=https://example.com/ > local.properties
```

```suggestion
      env:
        MAPBOX_KEY: ${{ secrets.MAPBOX_KEY }}
      run: |
        echo mapbox.key="$MAPBOX_KEY" > local.properties
        echo server.url=https://example.com/ > local.properties
```
Fancy multi-line yml syntax